internal/models/transformer_test.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var examplePubkey = Pubkey{
+	Name: "lzap-ed25519-2021",
+	Body: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap",
+}
+
+func TestPubkeyFingerprintGeneration(t *testing.T) {
```

Comment:
I don't have a strong feeling about this, but I feel like this is testing Pubkey transformation, not transform method itself...
But I guess two birds... just wondering if we add another transformation, would we test it all in this test file?

Reply:
Yeah I thought about this too, but the general practice is to create xyz_test.go for each xyz.go file. And it looks like you set the standard in validator to put all the code into the single file. Thus my intention is to keep this - all transformation functions to be in transformer.go thus the tests belong to transformer_test.go. I feel like this is okay since we only have a single function for each, we can break this down later.

Reply:
right, make sense, lets see how it goes once we actually add another transformations/validations :)